### ISSUE TITLE
add progressmeter

### ISSUE THREAD

**Issue by floswald** [floswald/MomentOpt.jl#24]
add https://github.com/timholy/ProgressMeter.jl

**Comment by JulienPascal** [comment]
I am really much willing to do this PR. ProgressMeter adds a really fancy touch to this package.
But should I wait until Edoardo's PR is merged?

**Comment by floswald** [comment]
Yes t depends on #25! Currently working on that. I have an issue with the jld2 part in fact.
